src/System.Private.DataContractSerialization/src/Resources/Strings.resx

```diff
@@ -1157,4 +1158,7 @@
   <data name="FailedToCreateMethodDelegate" xml:space="preserve">
     <value>Failed to create Delegate for method '{0}' of type '{1}'.</value>
   </data>
-</root>
+  <data name="ReadOnlyCollectionDeserialization" xml:space="preserve">
+    <value>Collection type '{0}' cannot be deserialized since it</value>
```

Comment:
truncated?

Reply:
Nope. I double and triple checked as I thought the same and this is correct. I think it's combined with another resource string describing what was being done at the time. This is used as a deferred exception message. Basically if you try to deserialize it throws an exception based on this message but the message is generated on creation of the serializer. It's combined with more context when you try to do an operation which causes an exception to be thrown with this message.

Reply:
interesting. I thought concatenation of segment fragments was not localization-friendly (?). But if it was that way already, it makes sense to leave it that way for servicing purposes.